src/Aspire.Hosting/Postgres/PostgresBuilderExtensions.cs

```diff
@@ -85,7 +85,7 @@ public static IResourceBuilder<PostgresDatabaseResource> AddDatabase(this IResou
                                   .WithAnnotation(new ContainerImageAnnotation { Image = "dpage/pgadmin4", Tag = "latest" })
                                   .WithHttpEndpoint(containerPort: 80, hostPort: hostPort, name: containerName)
                                   .WithEnvironment(SetPgAdminEnviromentVariables)
-                                  .WithVolumeMount(Path.GetTempFileName(), "/pgadmin4/servers.json")
+                                  .WithBindMount(Path.GetTempFileName(), "/pgadmin4/servers.json")
```

Comment:
Same as previous comment. What mount type should this be?

Reply:
This should be a bind mount as well.